.github/workflows/stale.yml

```diff
@@ -16,15 +16,16 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue is stale because it has been open for 60 days with no activity.'
+        stale-issue-message: 'This issue is stale because it has been open for 60 days with no activity. If the issue is still present, please leave a comment within 14 days to keep it open, otherwise it will be closed automatically.'
         close-issue-message: 'This issue was closed because it has been inactive for 14 days since being marked as stale.'
-        stale-pr-message: 'This pull request is stale because it has been open for 60 days with no activity.'
+        stale-pr-message: 'This pull request is stale because it has been open for 60 days with no activity. If the pull request is still valid, please update it within 14 days to keep it open or merge it, otherwise it will be closed automatically.'
         close-pr-message: 'This pull request was closed because it has been inactive for 14 days since being marked as stale.'
         stale-issue-label: 'stale'
         stale-pr-label: 'stale'
         days-before-stale: 60
         days-before-close: 14
         remove-stale-when-updated: true
         exempt-all-assignees: true
+        exempt-issue-labels: '📌 Pinned'
```

Comment:
`Pinned` is an actual Github's status, I suggest we use a different label to avoid confusion.

Reply:
Sure, changed in `keep open` in 750898e